content/200-concepts/100-components/02-prisma-client/053-client-extensions/140-shared-extensions.mdx

```diff
@@ -93,8 +97,14 @@ export default Prisma.defineExtension({
 
 </details>
 
+### Publishing the shareable extension to npm
+
 You can then share the extension on `npm`. When you choose a package name, we recommend that you use the `prisma-extension-<package-name>` convention, to make it easier to find and install.
 
+## Other considerations
+
+bla bla, other things to be aware or take advantage of
```

Comment:
this needs to be resolved it seems like, maybe just drop the entire section if we have no content for it?